File: events/welcome.js
module.exports = {
  name: 'guildMemberAdd',
  execute(member) {
    const channel = member.guild.channels.cache.find(ch => ch.name === 'ต้อนรับสมาชิก') 
                    || member.guild.systemChannel;

    if (!channel) return;
    channel.send(`🎉 ยินดีต้อนรับคุณ ${member} เข้าสู่เซิร์ฟเวอร์ครับ! อย่าลืมไปกดยืนยันตัวตนนะ ✨`);
  },
};

File: cogs/welcome.py
import discord
from discord.ext import commands


class Welcome(commands.Cog):

    def __init__(self,bot):
        self.bot=bot


    @commands.Cog.listener()
    async def on_member_join(self,member):

        channel = discord.utils.get(
            member.guild.text_channels,
            name="welcome"
        )

        if channel:

            embed=discord.Embed(
                title="ยินดีต้อนรับ",
                description=f"สวัสดี {member.mention}",
                color=0x00ffff
            )

            await channel.send(
                embed=embed
            )


async def setup(bot):
    await bot.add_cog(Welcome(bot))